test/decl/var/async_let.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %target-typecheck-verify-swift -enable-experimental-concurrency
+
+// REQUIRES: concurrency
+
+async let x = 1 // okay
```

Comment:
Curious: why allow non-async initializers?

Reply:
The work you want to run concurrently could be synchronous work. Why not?

Reply:
I didn't think that one through, sorry. I assumed (I don't know why) that concurrent code would be via async functions.